docs-ref-mapping/reference-unified.yml

```
@@ -676,6 +676,19 @@
       landingPageType: Service
       children:
       - 'azure-mgmt-scheduler*'
+  - name: Schema Registry
+      uid: azure.python.sdk.landingPage.services.SchemaRegistry
+      items:
+      - name: Schema Registry Client
+        uid: azure.python.sdk.landingPage.services.SchemaRegistry.Client
+        href: ~/docs-ref-services/schemaregistry-readme-pre.md
```

```suggestion
        href: ~/docs-ref-services/schemaregistry-readme.md
```

I know it feels counter-intuitive. 

1. I've got a work item to add more documentation about the why
2. I'll extend the script to automatically correct this.